acceptance/config/beaker/jenkins/redhat7-64m-64a.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
All of the other files in this directory have a ".cfg" extension and don't lead with the "---".  Not sure if it's more or less conventional for this to be a ".yaml" file instead of a ".cfg" file?